server/mailet/mailets/src/main/java/org/apache/james/transport/mailets/remote/delivery/DeliveryRunnable.java

```diff
@@ -47,7 +48,7 @@
 
     public static final Supplier<Date> CURRENT_DATE_SUPPLIER = Date::new;
     public static final AtomicBoolean DEFAULT_NOT_STARTED = new AtomicBoolean(false);
-    private static final String OUTGOING_MAILS = "outgoingMails";
+    public static final String OUTGOING_MAILS = "outgoingMails";
```

Comment:
Is package private enough?

Reply:
I want to re-use it in `DeliveryRunnableTest`